backend/danswer/llm/llm_provider_options.py

```diff
@@ -100,8 +100,8 @@ def fetch_available_well_known_llms() -> list[WellKnownLLMProviderDescriptor]:
             api_version_required=False,
             custom_config_keys=[],
             llm_names=fetch_models_for_provider(ANTHROPIC_PROVIDER_NAME),
-            default_model="claude-3-5-sonnet-20240620",
-            default_fast_model="claude-3-5-sonnet-20240620",
+            default_model="claude-3-5-sonnet-20241022",
```

Comment:
Watch out for context length issues - unless we bump our LiteLLM version (when they add "support" for this model), will use default max tokens (which is super low, something like `4096`)